design/KEPS/namespace-discovery/proposal.md

```diff
@@ -12,17 +12,18 @@ Provide a better way for namespace discovery within both server and operator.
 
 # Motivation
 
-Currently, the Kiali Server and Operator have many different ways to configure which namespaces should be accessible and viewable by the Kiali UI. The configurations are complex and confusing to users. We want to make things more easily configurable and understandable while still maintaining the ability for Kiali to have either cluster-wide access or limited access to specific namespaces.
+Currently, the Kiali Server and Operator have many different ways to configure which namespaces should be accessible and viewable by the Kiali UI. The configurations are complex and confusing to users. We want to make things more easily configurable and understandable while still maintaining the ability for Kiali to have either cluster-wide access or limited access to specific namespaces. We also want kiali to support soft multitenancy using the same mechanism as istio to make configuration more consistent with istio and easier for users.
```

Comment:
We need to define what we mean by "soft" multitenancy somewhere in here. Maybe even define "multitenancy" to differentiate what we mean.

Oh, and missing capitalization causes my OCD to flare up :)
```suggestion
Currently, the Kiali Server and Operator have many different ways to configure which namespaces should be accessible and viewable by the Kiali UI. The configurations are complex and confusing to users. We want to make things more easily configurable and understandable while still maintaining the ability for Kiali to have either cluster-wide access or limited access to specific namespaces. We also want Kiali to support soft multitenancy using the same mechanism as Istio to make configuration more consistent with Istio and easier for users.
```

Reply:
"soft" in this case means kiali still has cluster wide access but limits what it shows / operates on to the namespaces selected by discovery selectors.